features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/org.wso2.carbon.identity.core.server.feature.default.json

```diff
@@ -674,6 +674,12 @@
   "event.default_listener.identity_verification_event_listener.priority": "120",
   "event.default_listener.identity_verification_event_listener.enable": true,
 
+  "event.default_listener.identity_user_workflow_operation.enable": true,
+  "event.default_listener.identity_user_workflow_operation.priority": "10",
```

Comment:
Let's check whether we have conflicting priority numbers for listeners of
UserOperationEventListener and RoleManagementListener

Let's check for deployment.toml files of IDaaS type deployements also.